go/mysql/capabilities/capability.go

```diff
@@ -47,6 +47,7 @@ const (
 	CheckConstraintsCapability                                          // supported in MySQL 8.0.16 and above: https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-16.html
 	PerformanceSchemaDataLocksTableCapability                           // supported in MySQL 8.0.1 and above: https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-1.html
 	InstantDDLXtrabackupCapability                                      // Supported in 8.0.32 and above, solving a MySQL-vs-Xtrabackup bug starting 8.0.29
+	ReplicaQueries                                                      // Supported in 8.0.22 and above, using SHOW REPLICA STATUS and all variations.
```

Comment:
Nitty, but IMO this is better:
```
ReplicaTerminologyCapability                                                      // Both the `replica` and `slave` terminology are supported in 8.0.22 and above, e.g. SHOW REPLICA STATUS, with the `slave` terminology removed in 8.4.
```

Reply:
Also note that some of the terminology support came AFTER 8.0.22. For example: `log_replica_updates` was in 8.0.26+

Reply:
Fixing!